src/components/layout/header/AccountSwitcherWallet/account-switcher-wallet-item.tsx

```diff
@@ -59,6 +60,12 @@ export const AccountSwitcherWalletItem = observer(
 
             localStorage.setItem('authToken', token);
             localStorage.setItem('active_loginid', loginId.toString());
+            Analytics.setAttributes({
+                account_type: loginId
+                    .toString()
+                    .match(/[a-zA-Z]+/g)
+                    ?.join(''),
```

Comment:
nitpick:can we pass this to a variable or const and call here

Reply:
done